mmseg/models/decode_heads/setr_up_head.py

```diff
@@ -25,6 +25,7 @@ def __init__(self,
                  num_convs=1,
                  up_scale=4,
                  kernel_size=3,
+                 init_cfg=None,
```

Comment:
We can set the default `init_cfg` to implement the same weights initialization as that in `init_weights`

Reply:
And we can set it to `None` in configs if SETR does not use it.